docs/about/contributing.mdx

```diff
@@ -126,3 +126,44 @@ We're deeply thankful for the backing from the Open Collective, Open Source Coll
     By covering our hosting costs, Vercel directly contributes to Meshtastic's growth, enhancing our web scalability and presence. Their sponsorship is crucial for our continued development and focus on creating a robust platform for decentralized communication.
   </div>
 </div>
+
+## Contributing to the Meshtastic Blog
+
+### Purpose
+
+To share knowledge, updates, and innovations related to Meshtastic helping to enrich, engage, and grow the community.
+
+### Content Guidelines
+
+**Acceptable Topics**: Focus on Meshtastic-related themes, such as tutorials, use cases, development updates, and technology deep dives.
+**Unacceptable Topics**: Avoid self-promotion or commercial content. Promoting hardware, shops, cases, or other websites is unacceptable.
+**Author Biography**: You may link to your personal work or website. Maximum 250 characters. Photos must adhere to the [Meshtastic Conduct](https://meshtastic.org/docs/legal/conduct/) guidelines.
+
+### Rights and Posting
+
+**Content Originality**: Posts must be your original work but **can** be published elsewhere.
+**Content Rights**: Contributions are under the same CLA as Meshtastic’s other community contributions, aligning with the project’s open-source nature.
+
+### Submission and Review Process
+
+**How to Submit**: Articles must be submitted via Pull Requests to the [meshtastic/meshtastic](https://github.com/meshtastic/meshtastic) GitHub Repository.
+**Review Process**: Submissions will undergo a review for adherence to content guidelines, quality of information, and relevance to the community.
+
+### Frequency of Contributions
+
+**Posting Limits**: To ensure diversity, contributions are limited to two posts per month. Exceptions are available.
```

Comment:
I'm unsure about "posting limits".  A cap may not be necessary.  Would something like this make sense: "Authors posting more than 2 posts per month may be asked to space out their contributions to allow greater diversity" ?